include/libdnf/comps/group/package.hpp

```diff
@@ -35,16 +37,16 @@ enum class PackageType { MANDATORY, DEFAULT, OPTIONAL, CONDITIONAL };
 class Package {
 public:
     explicit Package(std::string name, PackageType type, std::string condition)
```

Comment:
Any non-basic type arguments (here the `std::string`s) should under normal circumstances always be passed by a const reference for the least amount of overhead. Then below, instead of moving, you just do a copy via `name(name)`. That's the canonical way to do this in C++.

```c++
   explicit Package(const std::string & name, PackageType type, const std::string & condition)
```

For completeness you could pass all arguments this way, the argument there is that a simple type is something up to 8B in size (on 64bit archs), so you're not saving anything by using a const reference which is of the same (pointer) size.

Reply:
Thanks, fixed.